source/mainnet/net/resources/glossary.rst

```diff
@@ -309,6 +330,29 @@ in concert with the identity provider.
 
 Initial accounts are only relevant for Desktop Wallet and |mw-gen1|.
 
+.. _glossary-initialize:
+
+Initialize
+==========
+
+Action that creates a new smart contract instance with the initial state. The only way to update the instance state is by invoking the instance’s entrypoints.
+
+.. _glossary-instance:
+
+Instance
+========
+
+A smart contract module together with a specific state and an amount of CCD tokens. Multiple smart contract instances can be created from the same module. Smart contract instances can be created from a deployed :ref:`smart contract
+module<contract-module>` via the ``init`` transaction which invokes the requested function in the smart contract module. This function can take a parameter.Its end result is required to be the initial smart contract state of the
```

Comment:
I'm confused by the formulation that the result is required to be the initial state. I thought the initial state is by definition the value returned by that function!?